February edition

equivalentideas opened the pull request:
This will go out Thursday 19th, Feb 2015.

Items:
- [x] closes #20 
- [x] closes #19 
- [x] closes #17 
- [x] closes #22
- [x] closes #24 
- [x] closes #25 
- [x] closes #26 
- [x] closes #27 
- [x] closes #28 
- [x] closes #29 
- [x] closes #30

equivalentideas committed:
add a blank .md file for the February edition

equivalentideas committed:
first draft of intro text

equivalentideas committed:
Revert "first draft of intro text"

This reverts commit 60e82fae5e4b1f106599720f14e7847cae4edbae.

equivalentideas committed:
add link and text for Percs
closes #17

equivalentideas committed:
add link and text for Civic Hall
closes #19

equivalentideas committed:
fix broken link formatting

equivalentideas committed:
add links and text for Open Data Day events
closes #24

equivalentideas committed:
Link and text for 12 steps
closes #25

equivalentideas committed:
add link and text for Hackfarming Blood Buddies
closes #26

equivalentideas committed:
remove double space

equivalentideas committed:
add link and text for 3 ways to keep They Vote For You ...
closes #27

equivalentideas committed:
add link and text for Git resources
closes #29

equivalentideas committed:
add link and text for Nick's article
closes #30

equivalentideas committed:
add link and text for Cuttlefish item
closed #22

equivalentideas committed:
add link and text for DTO item
closes #20

equivalentideas committed:
add link and text for Growstuff
closes #28

henare committed:
Fix typos

henare commented on editions/2015-02.md:
```diff
@@ -0,0 +1,61 @@
+
+
+### [12 steps toward a better neighbourhood](http://www.onbeing.org/program/feature/12-steps-toward-better-neighborhood/4723)
+Henare and Luke visited the [Friends of Erskineville](http://www.erskinevillevillage.org/) community group in Sydney last week. They introduced the group to [PlanningAlerts](https://www.planningalerts.org.au/) and had a great time finding out about how they’re working to impact their area. They picked up this list of *12 steps* that they were discussing. These are practical ideas for organising f ...
```
I don't really understand the last sentence here.

kat commented:
@equivalentideas I see what @henare is saying here, your description of why this is useful is circuitous. If I ask you directly that might help. So - Why have you included the  [12 steps toward a better neighbourhood](http://www.onbeing.org/program/feature/12-steps-toward-better-neighborhood/4723)? What _specifically_ is useful about it for the civic tech community, why is it applicable to them?

equivalentideas commented:
> circuitous

agreed.

> What specifically is useful about it for the civic tech community, why is it applicable to them?

I think this could give people ideas for things to work on.

Maybe:

> Maybe these steps will spark some ideas for new projects.

equivalentideas committed:
add introduction text

kat commented on editions/2015-02.md:
```diff
@@ -0,0 +1,61 @@
+
+### [Federal Government announces the Digital Transformation Office](https://www.pm.gov.au/media/2015-01-23/establishment-digital-transformation-office)
+
+The Government is establishing a “small team of developers, designers, researchers and content specialists working across government ... to make services simpler, clearer and faster for Australian families and businesses.” This is an initiative that the Australian civic tech community have been looking forward to since the establishment of the [Government Digital Services (GDS) office in the UK]( ...
```
'This is an initiative that the Australian civic tech community have been looking forward to since the establishment of the [Government Digital Services (GDS) office in the UK](https://www.gov.uk/). ' is longwinded, doesn't sounds like OAF, assumes that OAF speaks for a civic tech community.

equivalentideas replied:
What about just dropping that sentence?

kat replied:
Yes, that works. Then just remeber to write out Government Digital Services (GDS) office in full when you link off to their blog at the bottom.

equivalentideas replied:
> The UK’s Government Digital Services have also written a blog post about the project.

https://github.com/openaustralia/newsletter/blob/February2015/editions/2015-02.md#federal-government-announces-the-digital-transformation-office

kat commented on editions/2015-02.md:
```diff
@@ -0,0 +1,61 @@
+
+### [Local events for Internation Open Data Day](http://au.okfn.org/2015/02/10/australian-activities-on-international-open-data-day-21-february/)
+
+This Saturday, 21st Feb, is [International Open Data Day](http://opendataday.org/). The Open Knowledge Foundation Australia have done a wrap-up of events, including a day long workshopping session hosted by [Code for Australia](http://www.codeforaustralia.org/) in [Sydney tomorrow](http://www.eventbrite.com.au/e/international-open-data-day-sydney-tickets-15462067460). It is a free event and regis ...
```
so I think of a wrap up as being finishing something up or summarizing something which is complete - maybe that's a newer usage? also the use of 'have done' - past tense for a future event - I find it confusing. If I'm not alone in this - change "have done a wrap-up". Also let's change "a day long workshopping session" to "a day of workshops" or a "day long workshop"  if it is one long workshop? Also remove 'if you can make it' it's not needed.

equivalentideas replied:
fixed in https://github.com/openaustralia/newsletter/commit/0ed8bbef14310876e662ef2b7b2cdaa8ee1749c5

kat commented on editions/2015-02.md:
```diff
@@ -0,0 +1,61 @@
+Contributing through the Git version control system is an important skill for people working on civic tech and open source generally—but it ain't that easy to learn and it's not always clear where to start. We've observed that it can be a barrier to people getting involved.
+
+Aside from the link above there are many tutorials and articles to help. Luke found the [Web Ahead Podcast's Git episode](http://thewebahead.net/40) very helpful. Github also have a whole [list of resources](https://help.github.com/articles/good-resources-for-learning-git-and-github/). Sometimes the best thing to do is ask someone to sit down and take you through it.
+
```
"Aside from the link above - wordy! Instead of telling people how easy it is for them not to do something (just like everyone else) Try giving them something to go on, if that's the intent of providing the link of resources. "It's easier than you think to get going! Use one of the many useful tutorials at github.com, or ask people at your local _insert some event/online resource where people might be able to help one to one here_. If there is a pointer like that then the last sentence reinforces the "Sometimes the best thing to do..." and you might want to swap the last two sentences round in that case. 

The last sentence leaves an odd dangling conclusion. Does that agree with the article or is it an independent statement, and if so where might someone go to sit down and go through it? why is that there?

equivalentideas replied:
What about:

> Once you’re rolling, it really is easier than you’d think. You’ll quickly see why it’s so popular. If you learn well with audio, try the [Web Ahead Podcast's Git episode](http://thewebahead.net/40). If video, interactive tutorials or reading works better, try one of the many useful tutorials on [Github’s resources list](https://help.github.com/articles/good-resources-for-learning-git-and-github/).

Need a last sentence about asking for help.

equivalentideas replied:
I personally think that's important since that's how got over the initial hurdle :)

kat replied:
How about 
'Once you've learned it you'll have superpowers, so don't be afraid to ask if you need help' @henare your thoughts?

henare replied:
@kat like it!

equivalentideas replied:
love it. I'll add that now.

kat commented on editions/2015-02.md:
```diff
@@ -0,0 +1,61 @@
+
+We love this one because it’s a practical open source project helping people grow food and community. It helps keep information held in plants and seeds open by making it easy to share. Growstuff releases the data it collects under open licenses, giving power to it’s developers as well as growing a community of gardeners. It promotes pair programming as the norm.
+
+This is a very active project. Get involved as a volunteer to help test, write for [the Growstuff blog](http://blog.growstuff.org/), [contribute code to the application](https://github.com/growstuff) or just dig in, grow some food and share your seeds and data.
```
Get involved or just Volunteer - not both (I prefer Volunteer, more specific)

equivalentideas replied:
:+1:

equivalentideas replied:
Fixed in https://github.com/openaustralia/newsletter/commit/394536548e1f5e15e67b48a3e155c777adfacb8d

equivalentideas committed:
add link and text for Info_Aus interview

equivalentideas committed:
simplify sentence sentence

equivalentideas committed:
simplify text of open data day item

equivalentideas committed:
simplify text of FoE item

equivalentideas committed:
make git item more encouraging
using Kat's suggestion https://github.com/openaustralia/newsletter/pull/21#discussion_r24965511

equivalentideas committed:
shorten DTO item

equivalentideas committed:
remove broken link typo

henare committed:
Use my preferred name for the meetup

henare committed:
"Info Aus" => "InfoAus"

equivalentideas committed:
Use 'we' and 'they' consistently

equivalentideas committed:
fix typo in heading

henare commented:
[Thar she blows!](http://us8.campaign-archive1.com/?u=86c7f63554037aed044df4556&id=d967693c1d&e=6f8abba176)
